utils/fish/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2018 OpenWrt.org
```

Comment:
Do not assign the copyright to OpenWrt. Either remove this or add you own statement.

Reply:
Done, I deleted them.